composer.json

```diff
@@ -9,14 +9,15 @@
     ],
 
     "require": {
-        "php": ">=5.3.3",
-        "symfony/framework-bundle": "~2.1",
-        "symfony/form": "~2.1"
+        "php": ">=5.5.9",
+        "symfony/framework-bundle": "~2.7",
+        "symfony/form": "~2.7"
```

Comment:
This constraint of version is not allowing `3.0`. We should use `~2.7|~3.0` constraint.